frontend/package.json

```diff
@@ -22,6 +23,7 @@
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-redux": "^8.0.2",
+    "react-toastify": "^9.0.5",
```

Comment:
Is this necessary? I thought Chakra came with a toast
 You can uninstall it using `npm uninstall package name`